game.py

```diff
@@ -590,8 +640,10 @@ def handleMessage(self, player, message):
         elif message.startswith("maze:"):
             # someone has a different maze than us
             self._activity.update_alert('Connected', 'Maze shared!')
-            running_time, seed, width, height = map(lambda x: int(x),
-                                                    message[5:].split(",")[:4])
+            running_time, seed, width, height, hole = map(lambda x: int(x),
+                                                          message[5:]
+                                                          .split(","))
+            self.hole = True if hole else False
```

Comment:
This line seems very complicated.  Should this be `self.hole = bool(hole)` ?

Reply:
Yes, I have corrected that.